<launch>

    <!-- TAILORED MPC -->
    <node pkg="tailored_mpc" type="tailored_mpc_exec" name="tailored_mpc" output="screen">
        <rosparam file="$(find tailored_mpc)/params/tailored_mpc.yaml"/>
    </node>

    <node name="dyn_recfg_load_tailored_mpc" pkg="dynamic_reconfigure" type="dynparam" 
          args="load /tailored_mpc $(find tailored_mpc)/params/dyn_tailored_mpc.yaml" />

</launch>